CMakeLists.txt

```diff
@@ -108,14 +108,7 @@ endif()
 
 option(ENABLE_LIBINTL "enable libintl" ON)
 option(ENABLE_LIBICONV "enable libiconv" ON)
-if (MINGW)
-  # Disable LTO by default as it may not compile
-  # See https://github.com/Alexpux/MINGW-packages/issues/3516
-  # and https://github.com/neovim/neovim/pull/8654#issuecomment-402316672
-  option(ENABLE_LTO "enable link time optimization" OFF)
-else()
```

Comment:
Sorry for the runaround, but we might want to wait on removing mingw stuff from CMakeLists. That's different that build.ps1, which is merely a "quickstart" script. If we remove mingw things from CMakeLists, that makes it very hard for anyone to build with mingw, if they wish.

I'm open to objections though.

Reply:
Yeah, I was about to ask you about this as well. Makes sense,